### PR TITLE
Api Response Update

### PR DESCRIPTION
**Pull request by nothingalike**
## Main Objectives
 - Gain access to Response Headers
 - Add `Prefer` Request Header 

### Gain access to Response Headers
Instead of returning just the payload, endpoints will pass through the response. Add `.Content` to your response to get the payload now. 

### Add `Prefer` Request Header 
If you include `prefer: PreferValues.CountEstimated`, your response will include the `Content-Range` Response Header. This Header's value will be something like this `0-4/20`
 - `0-4` this is the range of your items. this had `offset=0` and `limit=5` 
 - `20` this is the estimated total set of data. can assist with paging ui components

### CODE REVIEW

**Comment by nothingalike** [comment]
## Main Objectives
 - Gain access to Response Headers
 - Add `Prefer` Request Header 

### Gain access to Response Headers
Instead of returning just the payload, endpoints will pass through the response. Add `.Content` to your response to get the payload now. 

### Add `Prefer` Request Header 
If you include `prefer: PreferValues.CountEstimated`, your response will include the `Content-Range` Response Header. This Header's value will be something like this `0-4/20`
 - `0-4` this is the range of your items. this had `offset=0` and `limit=5` 
 - `20` this is the estimated total set of data. can assist with paging ui components

**Review by safestak-keith — APPROVED** [review]
Just seeking clarification around the parameters for IAccountClient.GetStakeAssets `asset_name` and `asset_policy`

File: CardanoSharp.Koios.Sdk/IAccountClient.cs [hunk 1]
```diff
@@ -7,25 +7,47 @@ namespace CardanoSharp.Koios.Sdk
...
-        Task<StakeAddress[]> GetStakeAddresses([AliasAs("_address")] string address, int? limit = null, int? offset = null);
+        Task<ApiResponse<StakeAddress[]>> GetStakeAddresses([AliasAs("_address")] string address, 
+            [AliasAs("limit")]int? limit = null, 
+            [AliasAs("offset")]int? offset = null, 
+            [Header("Prefer")] string prefer = null);
 
         [Get("/account_assets")]
-        Task<StakeAsset[]> GetStakeAssets([AliasAs("_address")] string address, [AliasAs("asset_name")] string assetName = null, [AliasAs("asset_policy")] string assetPolicy = null, int? limit = null, int? offset = null);
+        Task<ApiResponse<StakeAsset[]>> GetStakeAssets([AliasAs("_address")] string address, 
+            [AliasAs("asset_name")] string assetName = null,
```
[the remark this answers is not in the document]
Comment (nothingalike) [reply CardanoSharp.Koios.Sdk/IAccountClient.cs:41]: that is correct and it is case sensitive. example: i was using the `like` filter and had an NFT called `Blob #1` and had #2, #3 and #4. when filtering with `like`, i had to do this `like.*Blob*`. if the "B" was lowercase it did not find my assets

[the remark this answers is not in the document]
Comment (rdlrt) [reply CardanoSharp.Koios.Sdk/IAccountClient.cs:41]: > if the "B" was lowercase it did not find my assets

`like` works like native SQL, for case insensitive you'd use `ilike`. The table referred is a subset of commonly used operators, the complete list can be found [here](https://postgrest.org/en/stable/api.html#horizontal-filtering-rows) - it's referred at [API Usage](https://api.koios.rest/#overview--api-usage) section but we should prolly link a reference before the table.

**Pull request merged**